tests/product/test_installer.py

```diff
@@ -42,30 +42,12 @@ def tearDown(self):
         self.centos_container.tear_down()
 
     @attr('smoketest')
-    def test_online_installer(self):
-        self.pa_installer._build_installer_in_docker(self.centos_container,
-                                                     online_installer=True,
-                                                     unique=True)
+    def test_installer(self):
         self.__verify_third_party_dir(False)
         self.pa_installer.install(
             dist_dir=self.centos_container.get_dist_dir(unique=True))
         self.run_prestoadmin('--help', raise_error=True)
 
-    @attr('smoketest', 'offline_installer')
-    def test_offline_installer(self):
```

Comment:
I think it should still be run in automation, but it's fine if it's not in smoketest. 

Also, which gets run if you have both the online and offline installer in the dist directory? That's something we need to be careful of when running the tests.